common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolver.java

```diff
@@ -214,7 +219,9 @@ public ManagedBuffer getRddBlockData(
    * this method.
    */
   public void applicationRemoved(String appId, boolean cleanupLocalDirs) {
-    logger.info("Application {} removed, cleanupLocalDirs = {}", appId, cleanupLocalDirs);
+    logger.info("Application {} removed, cleanupLocalDirs = {}",
+      MDC.of(LogKeys.APP_ID$.MODULE$, appId),
+      MDC.of(LogKeys.LOCAL_DIRS$.MODULE$, cleanupLocalDirs));
```

Comment:
cleanupLocalDirs is a boolean. let's just use `CLEANUP_LOCAL_DIRS`

Reply:
Updated